engine/src/flutter/shell/platform/android/test/io/flutter/embedding/android/SurfaceHolderCallbackCompatTest.java

```diff
@@ -0,0 +1,118 @@
+package io.flutter.embedding.android;
```

Comment:
Can you point to some log or evidence in ci that this test is being run? We have missed that step in the past and I want to make sure we are not assuming that *Test.java files are executed.